daemon/api.go

```diff
@@ -254,46 +254,67 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 	}
 	defer lock.Unlock()
 
-	sources := make([]string, 1, 2)
-	sources[0] = "local"
-	// we're not worried if the remote repos error out
-	found, _ := newRemoteRepo().All()
-	if len(found) > 0 {
-		sources = append(sources, "store")
+	results := make(map[string]map[string]interface{})
```

Comment:
Yeah, it is confusing, it's a map of snap names to key-value pairs of snap data. I'll add a comment.